server/libs/template.js

```diff
@@ -179,6 +179,107 @@ const getYesOrNo = boolean => (
   boolean ? 'Yes' : 'No'
 );
 
+
+const shift = (number, precision) => {
+  const numArray = (`${number}`).split('e');
+  return +(`${numArray[0]}e${(numArray[1] ? (+numArray[1] + precision) : precision)}`);
+};
+
+const round = (number, precision) => (
+  shift(Math.round(shift(number, +precision)), -precision)
+);
+
+/**
+ * Round the float to 1 decimal
+ *
+ * @param {float} number
+ * @returns the rounded float number
+ */
+export const roundTo1Decimal = number => (
+  round(number, 1)
+);
+
+/**
+ *
+ * @param {number} numberOfAnimals
+ * @param {number} totalDays
+ * @param {number} auFactor parameter provided from the livestock type
+ * @returns {float} the total AUMs
+ */
+export const calcTotalAUMs = (numberOfAnimals = 0, totalDays, auFactor = 0) => (
+  ((numberOfAnimals * totalDays * auFactor) / 30.44)
```

Comment:
30.44 is a magic number. Define it in constants with a name or comment that will help describe it.